DEP Tasks/Trainers Task/JSCore/Task 1/Basic Mathematical Operations.js

```diff
@@ -0,0 +1,15 @@
+function basicOp(operation, value1, value2)
+{
+  // Code
+  if(operation==='+')
```

Comment:
For every conditions there should be '{ }' braces.

Reply:
done